kotor Randomizer 2/Randomization/RandoForm.cs

```diff
@@ -87,6 +87,10 @@ private void RunRando()
                 // Randomize the categories.
                 try
                 {
+                    // Copy over custom appearance.2da to fix potential model and texture issues.
```

Comment:
It is a little bit strange that we're adding this file regardless of rando settings, though I suppose it won't hurt anything. Though in the future as more of these "default" files come along, it may be advisable to compartmentalize their additions into a single method.